api/privileges.proto

```diff
@@ -0,0 +1,281 @@
+
+		/* Allow raising priority and setting priority on other (different
+		UID) processes */
+		/* Allow use of FIFO and round-robin (realtime) scheduling on own
+		processes and setting the scheduling algorithm used by another
+		process. */
+		/* Allow setting cpu affinity on other processes */
+		CAP_SYS_NICE          = 23;
+
+		/* Override resource limits. Set resource limits. */
+		/* Override quota limits. */
+		/* Override reserved space on ext2 filesystem */
+		/* Modify data journaling mode on ext3 filesystem (uses journaling
+		resources) */
+		/* NOTE: ext2 honors fsuid when checking for resource overrides, so
+		you can override using fsuid too */
+		/* Override size restrictions on IPC message queues */
+		/* Allow more than 64hz interrupts from the real-time clock */
+		/* Override max number of consoles on console allocation */
+		/* Override max number of keymaps */
+		CAP_SYS_RESOURCE      = 24;
+
+		/* Allow manipulation of system clock */
+		/* Allow irix_stime on mips */
+		/* Allow setting the real-time clock */
+		CAP_SYS_TIME          = 25;
+
+		/* Allow configuration of tty devices */
+		/* Allow vhangup() of tty */
+		CAP_SYS_TTY_CONFIG    = 26;
+
+		/* Allow the privileged aspects of mknod() */
+		CAP_MKNOD             = 27;
+
+		/* Allow taking of leases on files */
+		CAP_LEASE             = 28;
+
+		/* Allow writing the audit log via unicast netlink socket */
+		CAP_AUDIT_WRITE       = 29;
+
+		/* Allow configuration of audit via unicast netlink socket */
+		CAP_AUDIT_CONTROL     = 30;
+		CAP_SETFCAP           = 31;
+
+		/* Override MAC access.
+		The base kernel enforces no MAC policy.
+		An LSM may enforce a MAC policy, and if it does and it chooses
+		to implement capability based overrides of that policy, this is
+		the capability it should use to do so. */
+		CAP_MAC_OVERRIDE      = 32;
+
+		/* Allow MAC configuration or state changes.
+		The base kernel requires no MAC configuration.
+		An LSM may enforce a MAC policy, and if it does and it chooses
+		to implement capability based checks on modifications to that
+		policy or the data required to maintain it, this is the
+		capability it should use to do so. */
+		CAP_MAC_ADMIN         = 33;
+
+		/* Allow configuring the kernel's syslog (printk behaviour) */
+		CAP_SYSLOG            = 34;
+
+		/* Allow triggering something that will wake the system */
+		CAP_WAKE_ALARM        = 35;
+
+		/* Allow preventing system suspends */
+		CAP_BLOCK_SUSPEND     = 36;
+
+		/* Allow reading the audit log via multicast netlink socket */
+		CAP_AUDIT_READ        = 37;
+	}
+
+	message Device {
+		// The path to the device
+		string path = 1;
+		// The Read/Write/Mknod access for the device
+		string rwm = 2;
+	}
+
+	Version version = 1;
+
+	// Linux capabilities required by the container
+	repeated Capabilities capabilities = 2;
+
+	// Devices needed by the service
+	repeated Device devices = 3;
+
+	// Rather than provide a list of all devices, just mount every device on the node
```

Comment:
If this is the equivalent of `allow-all-devices`, then the comment should read `allow read write and modify access to all devices on the node`.